core/src/tpu.rs

```diff
@@ -222,14 +255,16 @@ impl Tpu {
 
         Self {
             fetch_stage,
-            sigverify_stage,
-            vote_sigverify_stage,
+            udp_sigverify_stage,
```

Comment:
can we do this renaming in a separate PR?

Reply:
Just to be clear, you're proposing we push the separate UDP vs. QUIC stage threads (but still keep unbounded channels as the packet passing mechanism) first as a separate PR, right?

Reply:
I'll just move this commit to a separate PR? https://github.com/solana-labs/solana/pull/25192/commits/16a3b4a96e59979563880785d236e7e31c911d12

Reply:
done as #25285

Reply:
ah, sorry I didn't realize it's adding those new ones.. why do we need them?

Reply:
We don't stricly need them. But when talking with @brennanwatt we realized that using one stage for QUIC and UDP would mean that a flood of UDP packets could crowd out QUIC packets. But QUIC packets are probably higher value. So we're now merging the flows only after sigverify.

If you like I can drop the new stages and instead merge in the queue leading to findpacketsenderstake.

Reply:
Re-combined the UDP/QUIC TPU stages